rasa/core/policies/memoization.py

```diff
@@ -226,7 +226,8 @@ def predict_action_probabilities(
 
         tracker_as_states = self.featurizer.prediction_states([tracker], domain)
         states = tracker_as_states[0]
-        logger.debug(f"Current tracker state {states}")
+        current_states = ' '.join(self.format_tracker_states(states))
```

Comment:
I think it makes sense to directly return a string, so that you don't need to call `" ".join()`

Reply:
Done it ?! We still using `.join()` but now it has been used in the return of the method.

Reply:
yes, thank you. That looks good, and seems to simplify the usage of this method